src/main/java/io/confluent/copycat/hdfs/CommittedFileFilter.java

```diff
@@ -17,8 +17,12 @@
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.fs.PathFilter;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 
 public class CommittedFileFilter implements PathFilter {
+  private static Pattern pattern = Pattern.compile("\\w+_\\d+_\\d+_\\d+");
```

Comment:
I can never remember the exact set of legal chars for topics, but I don't think `\w` will cover it. For example `-` should be a valid char I think. If you check in Kafka's code, I think there's a unit test that checks valid names, that's probably a good source to get the definitive list of valid characters.